Makefile

```diff
@@ -1,5 +1,5 @@
 PREFIX = /usr/local
-CFLAGS += -Wall -g
+CFLAGS += -Wall -g -DB460800 -DB921600 -DTERMIOS_TIMEOUT_MS=500
 
```

Comment:
I agree, this was left here after our experiments. I'll remove it. 

What about speed opts? Is it needed to have those defines  in code and here? Why not enable it by default? Driver won't let you set unsupported speed.